src/components/Ui/Blockie.vue

```diff
@@ -0,0 +1,19 @@
+<script setup>
+import { computed } from 'vue';
+
+import makeBlockie from 'ethereum-blockies-base64';
+const props = defineProps({
+  seed: {
+    type: String,
+    default: 'randomString'
+  }
+});
+
+const blockie = computed(() => {
```

Comment:
This could be simplified to this
```suggestion
const blockie = computed(() => makeBlockie(props.seed));
```